synth-converter/src/rdf/rdf_serializers.rs

```diff
@@ -0,0 +1,56 @@
+use crate::graph::prefix_map::generate_prefix_map;
+use anyhow::{Context, Result};
+use sophia::{
+    api::{
+        prelude::*,
+        serializer::{Stringifier, TripleSerializer},
+    },
+    inmem::graph::LightGraph,
+    jsonld::{serializer::JsonLdSerializer, JsonLdOptions},
+};
+use sophia_turtle::serializer::turtle::{TurtleConfig, TurtleSerializer};
+
+/// Serialize an RDF graph to Turtle format
+///
+/// # Parameters
+/// - `graph`: A reference to the graph to be serialized.
+///
+/// # Returns
+/// A `Result` containing the Turtle serialization as a `String`, or an error if serialization fails.
+pub fn serialize_graph_to_turtle(graph: &LightGraph) -> Result<String> {
```

Comment:
suggestion: You could write this function generic taking a `impl Write` 

https://doc.rust-lang.org/std/io/trait.Write.html 

such that you can pass an open file etc.